### ISSUE TITLE
Error when installing via TPM

### ISSUE THREAD

**Issue by filipeyay** [egel/tmux-gruvbox#46]
Whenever I try to install via TPM, it takes a long time until it gives an error.
Doing the manual installation (copying the .conf file) works normally.

![errorgruvboxtmuxtpm](https://github.com/user-attachments/assets/169f9266-1f5f-4c07-bc64-cee40d266f7f)

**Comment by egel** [comment]
Hi @filipeyay thanks for your time and reporting it! :) 

Strangely, you experienced this error. The current main branch should be stable for quite a while - at least I tried to do my best to make it rock solid in the major releases. 

Could you provide some additional information about your setup (full tmux config, os, shell) so I would have the opportunity to reproduce this? Additionally, you could also write each step you did (just for info) so I could better understand.

**Comment by egel** [comment]
I've also tried to remove the tmux-gruvbox plugin and reinstall it. Plugin seems to correctly downloaded - at least with the setup I use [.tmux.conf](https://github.com/egel/dotfiles/blob/main/configuration/.tmux.conf) (sidenote: this config is for tmux-gruvbox v2 I am currently testing, and which I will roll out soon - nevertheless configs should be fully compatible)

<img width="489" alt="image" src="https://github.com/user-attachments/assets/60f58bc7-5ce6-43d3-8a79-2426587522a7">

Maybe this error could be related to your tmux/shell/system configuration. Have you maybe checked this ticket [plugin "download failed"](https://github.com/tmux-plugins/tpm/issues/62) on tmux repo?

**Comment by filipeyay** [comment]
Hi @egel!

I'm using Ubuntu WSL2 with Zsh.

My full tmux config when I got this error:

```
# tpm
set -g @plugin 'tmux-plugins/tpm'
set -g @plugin 'tmux-plugins/tmux-sensible'

# gruvbox
set -g @plugin 'egel/tmux-gruvbox'
set -g @tmux-gruvbox 'dark-transparent'

# end
run '~/.tmux/plugins/tpm/tpm'
```

I'll read through the ticket you provided, thanks!

**Comment by filipeyay** [comment]
So, I tried deleting the TPM completely, and reinstalling it. I tried deleting all plugins and updating the installation, and I tried using your configuration file. They all gave the same error... About cloning the repository via https, this works normally.

I don't know if this helps but when I get to the "installing tmux-gruvbox" screen, I see that the directory is created in the plugins folder, but no data is ever written inside.

![Captura de tela 2024-11-27 005810](https://github.com/user-attachments/assets/7981f443-3616-4789-8f42-e00db6c68e4c)
